src/main.cpp

```diff
@@ -674,41 +674,43 @@ int main(int argc, char* argv[])
 		return 0;
 	}
 
-	if (argc == 2) {
-		start_viewer(argv[1]);
-	}
-
-	if (cli.bspfile.empty()) {
-		logf("ERROR: no map specified\n"); return 1;
-	}
-
 	if (cli.hasOption("-v")) {
 		g_verbose = true;
 	}

```

Comment:
This can also be moved to the `else` clause, since it would be ignored otherwise (`argc == 2` != `bspguy map.bsp -v`)

Reply:
yes

Reply:
Fixed